.github/workflows/ci.yml

```diff
@@ -152,9 +153,22 @@ jobs:
           </configurable_settings>
           EOF
 
+          # Create empty `schroot_env` file to suppress `shellcheck` error:
+          # SC1091: Not following: /tmp/schroot_env: openBinaryFile: does not exist (No such file or directory)
+          touch /tmp/schroot_env
```

Comment:
Could you please explain where is this required? I'd like to ask if we can remove the dependency on this directory existence, but I can't find any references to it outside of `lmi_setup_*.sh` scripts.

Reply:
It used from `lmi_setup_10c.sh`.

The error message:
```
In /home/runner/work/lmi/lmi/lmi_setup_10c.sh line 25:
. /tmp/schroot_env
  ^--------------^ SC1091: Not following: /tmp/schroot_env: openBinaryFile: does not exist (No such file or directory)
```

Reply:
If you rebase this on the latest master, you shouldn't need this any more (see 6644f9917a5d4622be8d6ed45329c1a76473de59) -- could you please check it?